robot/basestation/static/css/themes/mantis.css

```diff
@@ -0,0 +1,202 @@
+/*
+Bootstrap Buttons (Blue): #528EA5
+Regular Buttons (Purple): #656F8C
+Table (Purple): #675B79
+Table (Light Grey): #EFEFEF
+Navbar (Green): #6E9389
+Text (Black): #252931
+Background (Beige): #F2F1E8
+Button Hover (Blueish Light Grey): #BCD4DD
+*/
+
+/* Background color */
+html, body {
+  height: 100%;
+  background-color: #F2F1E8;
+}
+
+/* Navigation bar color */
+.navbar {
+  background-color: #6E9389;
+}
+
+/* Whole lotta bootstrap dropdown menu styling */
+.dropdown .nav-link {color:#528EA5; text-decoration: none;}
+.dropdown .btn {background: #528EA5; color:#fff;}
+.container .dropdown .dropdown-menu a:hover {
+  color: #fff;
+  background-color: #fff;
+  border-color: white;
+}
+.btn-primary.dropdown-toggle {
+    color: #fff;
+    border-color: #fff;
+}
+.bootstrap-select .dropdown-toggle:focus {
+  outline: none!important;
+  box-shadow:
+  0 0 0 0.2rem rgba(0,0,0,.5);
+}
+.btn-primary.focus, .btn-primary:focus {
+  box-shadow:
+  0 0 0 0.2rem rgba(255,255,255,0);
+}
+.btn-primary:not(:disabled):not(.disabled).active, .btn-primary:not(:disabled):not(.disabled):active, .show>.btn-primary.dropdown-toggle {
+  color: #fff;
+  background-color: #528EA5;
+  border-color: #528EA5;
+}
+.btn-primary:not(:disabled):not(.disabled).active:focus, .btn-primary:not(:disabled):not(.disabled):active:focus, .show>.btn-primary.dropdown-toggle:focus {
+  box-shadow: 0 0 0 0.2rem rgba(255,255,255,0);
+}
+.btn.focus, .btn:focus {
+  box shadow: white;
+}
+.btn-primary {
+  border-color: white;
+}
+.dropdown-item.active, .dropdown-item:active {
+    color: #fff;
+    text-decoration: none;
+    background-color: #BCD4DD;
+}
+
+/* Bootstrap buttons */
+.btn-primary {
+  color: #fff;
+  background-color: #528EA5;
+  border-color: #528EA5;
+}
+.btn-primary:hover {
+  color: black;
+  background-color: #BCD4DD;
+  border-color: #BCD4DD;
+}
+
+/* Bootstrap forms */
+.form-control:focus {
+  color: white;
+  background-color: #fff;
+  border-color: white;
+  outline: 0;
+  box-shadow: 0 0 0 0.2rem rgba(255,255,255,0);
+}
+
+/* Titles */
+.container-header, thead {
+  color: #252931;
+  font-weight: bolder;
+}
+.title {
+  color: #252931;
+}
+
+/*Log buttons (rover.css)*/
+#logs-btn button {
+    background-color: #656F8C;
+    border: ridge 2px black;
+}
+
+/* Control Buttons (rover.css) */
+.control-btn {
+  background-color: #656F8C;
+  border: ridge 2px black;
+  color: white;
+}
+.control-btn2 {
+    background-color: #656F8C;
+    border: ridge 2px black;
+    color: white;
+}
+.control-btn3 {
+    background-color: #656F8C;
+    border: ridge 2px black;
+    color: white;
+}
+.control-btn4 {
+    background-color: #656F8C;
+    border: ridge 2px black;
+    color: white;
+}
```

Comment:
You should put only 1 class for this whole thing.
something like ".control-btn"

If there's logic going on with the control buttons they should have a **descriptive ID**

Reply:
Changed the IDs